src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -33,6 +33,9 @@ public class BicepCompletionProvider : ICompletionProvider
     {
         private const string MarkdownNewLine = "  \n";
 
+        private const int MaxHoverMarkdownCodeBlockLength = 90000;
```

Comment:
Rather than duplicating this property and the `CodeBlock` / `CodeBlockWithDescription` methods, I think it would be worth creating a static helper class to share the code across both completion & hover handlers (`MarkdownHelper`?)